### PR TITLE
Release sdk b020

### PR DESCRIPTION
**Pull request by bobbytreed**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.1 document
- [ ] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[developer/hosting/runspace02-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace02-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace03-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace03-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace04-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace04-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace05-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace05-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace06-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace06-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace07-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace07-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace08-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace08-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace09-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace09-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace10-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace10-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace11-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace11-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/windows-powershell-api-samples.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/windows-powershell-api-samples.md) | :white_check_mark:Succeeded | |
[developer/hosting/windows-powershell-host-quickstart.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/windows-powershell-host-quickstart.md) | :white_check_mark:Succeeded | |
[developer/hosting/windows-powershell01-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/windows-powershell01-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/windows-powershell02-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/windows-powershell02-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/writing-a-windows-powershell-host-application.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/writing-a-windows-powershell-host-application.md) | :white_check_mark:Succeeded | |
[developer/module/file-types-permitted-in-an-updatable-help-cab-file.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/file-types-permitted-in-an-updatable-help-cab-file.md) | :white_check_mark:Succeeded | |
[developer/module/helpinfo-xml-sample-file.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/helpinfo-xml-sample-file.md) | :white_check_mark:Succeeded | |
[developer/module/helpinfo-xml-schema.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/helpinfo-xml-schema.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-create-a-helpinfo-xml-file.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-create-a-helpinfo-xml-file.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-create-and-upload-cab-files.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-create-and-upload-cab-files.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-name-a-helpinfo-xml-file.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-name-a-helpinfo-xml-file.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-name-an-updatable-help-cab-file.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-name-an-updatable-help-cab-file.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-prepare-updatable-help-cab-files.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-prepare-updatable-help-cab-files.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-set-helpinfo-xml-version-numbers.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-set-helpinfo-xml-version-numbers.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-test-updatable-help.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-test-updatable-help.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-update-help-files.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-update-help-files.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C8%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806081555040940-2538%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=s0eI9ViUvxI6%2ByyzmfDOr%2Bq3T6Bg2ApZrTSvxwktw1Q%3D&st=2018-06-08T16%3A00%3A07Z&se=2018-07-09T16%3A05%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: developer/module/helpinfo-xml-sample-file.md [hunk 1]
```diff
@@ -1,42 +1,44 @@
...
+      <UICultureVersion>1.1.0.5</UICultureVersion>
+    </UICulture>
+    <UICulture>
+      <UICultureName>ja-JP</UICultureName>
+      <UICultureVersion>3.2.0.4</UICultureVersion>
+    </UICulture>
+   </SupportedUICultures>
+</HelpInfo>
+
+```
```
Comment (sdwheeler) [inline developer/module/helpinfo-xml-sample-file.md]: Control character

File: developer/hosting/windows-powershell-host-quickstart.md [hunk 2]
```diff
@@ -1,162 +1,172 @@
...
+       Console.WriteLine(entry.Name);
+}
+```
+
+When run, the output of this code will look as follows.
+
+```powershell
+Get-Command
+Import-Module
+```
```
Comment (sdwheeler) [inline developer/hosting/windows-powershell-host-quickstart.md]: Control character

File: developer/module/helpinfo-xml-sample-file.md (hunk 1 above)
Comment (bobbytreed) [reply developer/module/helpinfo-xml-sample-file.md]: Fixed this

File: developer/hosting/windows-powershell-host-quickstart.md (hunk 2 above)
Comment (bobbytreed) [reply developer/hosting/windows-powershell-host-quickstart.md]: Fixed this

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[developer/hosting/runspace02-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace02-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace03-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace03-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace04-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace04-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace05-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace05-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace06-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace06-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace07-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace07-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace08-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace08-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace09-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace09-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace10-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace10-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/runspace11-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/runspace11-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/windows-powershell-api-samples.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/windows-powershell-api-samples.md) | :white_check_mark:Succeeded | |
[developer/hosting/windows-powershell-host-quickstart.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/windows-powershell-host-quickstart.md) | :white_check_mark:Succeeded | |
[developer/hosting/windows-powershell01-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/windows-powershell01-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/windows-powershell02-sample.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/windows-powershell02-sample.md) | :white_check_mark:Succeeded | |
[developer/hosting/writing-a-windows-powershell-host-application.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/hosting/writing-a-windows-powershell-host-application.md) | :white_check_mark:Succeeded | |
[developer/module/file-types-permitted-in-an-updatable-help-cab-file.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/file-types-permitted-in-an-updatable-help-cab-file.md) | :white_check_mark:Succeeded | |
[developer/module/helpinfo-xml-sample-file.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/helpinfo-xml-sample-file.md) | :white_check_mark:Succeeded | |
[developer/module/helpinfo-xml-schema.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/helpinfo-xml-schema.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-create-a-helpinfo-xml-file.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-create-a-helpinfo-xml-file.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-create-and-upload-cab-files.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-create-and-upload-cab-files.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-name-a-helpinfo-xml-file.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-name-a-helpinfo-xml-file.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-name-an-updatable-help-cab-file.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-name-an-updatable-help-cab-file.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-prepare-updatable-help-cab-files.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-prepare-updatable-help-cab-files.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-set-helpinfo-xml-version-numbers.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-set-helpinfo-xml-version-numbers.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-test-updatable-help.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-test-updatable-help.md) | :white_check_mark:Succeeded | |
[developer/module/how-to-update-help-files.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-sdk-b020/developer/module/how-to-update-help-files.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C8%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806081743393954-2538%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=tTVtupJtCaeVYyYoMQszP8iGeFPYmGBL8ytomRQKfOI%3D&st=2018-06-08T17%3A48%3A18Z&se=2018-07-09T17%3A53%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**